src/commands/basemaps-github/make.cog.github.ts

```diff
@@ -296,19 +306,19 @@ export class MakeCogGithub {
   async prepareElevationTileSetConfig(layer: ConfigLayer, tileSet?: ConfigTileSetRaster): Promise<ConfigTileSetRaster> {
     if (tileSet == null) {
       // Prepare individual elevation tileset config
-      const targetLayer = { ...layer, Category: Category.Elevation, minZoom: 0, maxZoom: 32 };
+      const targetLayer = { ...layer, Category: 'Elevation', minZoom: 0, maxZoom: 32 };
```

Comment:
@Wentao-Kuang I noticed this had the wrong name, it should be `category` with a lower case `c` anythoughts on the impact by fixing it?

Reply:
I think that should be lower case, please fix it. And lucky we don't have individual Elevation tiles so far.